includes/admin/meta-boxes/views/html-related-orders-table.php

```diff
@@ -22,7 +23,31 @@
 			</tr>
 		</thead>
 		<tbody>
-			<?php do_action( 'woocommerce_subscriptions_related_orders_meta_box_rows', $post ); ?>
+			<?php
+			if ( has_action( 'woocommerce_subscriptions_related_orders_meta_box_rows' ) ) {
+				wcs_deprecated_hook( 'woocommerce_subscriptions_related_orders_meta_box_rows', 'subscriptions-core 5.1.0', 'wcs_related_orders_meta_box_rows' );
+
+				/**
+				 * Renders renewal order rows in the Related Orders table.
+				 *
+				 * This action is deprecated in favour of 'wcs_related_orders_meta_box_rows'.
+				 *
+				 * @deprecated subscriptions-core 5.1.0
+				 *
+				 * @param WC_Post $post The order post object.
+				 */
+				do_action( 'woocommerce_subscriptions_related_orders_meta_box_rows', wc_get_order( $post ) );
```

Comment:
is this suppose to just be `$post`? 
```suggestion
				do_action( 'woocommerce_subscriptions_related_orders_meta_box_rows', $post );
```

Reply:
Yep! Thanks, resolved in 1881951